### DISCUSSION TITLE
changing keyboard shortcuts of alt+1 ~ alt+9

### DISCUSSION THREAD

**Discussion by zangetsudo** [kingToolbox/WindTerm#442] [Ideas]
current it's default to menu, we do like to use it to switch tabs.
is it possible? 
do we need to re-compile the app???

**Comment by gooker** [comment]
<img width="450" alt="1642042790(1)" src="https://user-images.githubusercontent.com/5688076/149258041-e8e94ea3-8546-44c9-b6f4-ae884dd58147.png">

**Reply by zangetsudo** [reply]
hi, gooker
we have tried to change the default key mapping, say map alt+1 ~ alt+9 to tabs, we can switch the tab by press alt+N twice which is not what we want.

i dont understand what your pic mean anyway.

regards.

**Reply by gooker** [reply]
1,
<img width="437" alt="1642125309(1)" src="https://user-images.githubusercontent.com/5688076/149258041-e8e94ea3-8546-44c9-b6f4-ae884dd58147.png">

2,
<img width="437" alt="1642125309(1)" src="https://user-images.githubusercontent.com/5688076/149438086-a6f01ab4-b93b-4259-826e-2df72d3b4ee9.png">

**Reply by zangetsudo** [reply]
hi, gooker

after a little change with your post, we can do what we want to switch tabs.
thank you very much.

regards.
